src/app/core/hq-dashboard-sub-menu/hq-dashboard-sub-menu.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, Input, OnInit } from '@angular/core';
 import { Router, NavigationEnd, ActivatedRoute } from '@angular/router';
 import { SocialMediaType } from '../../view-models/social-media-type';
 import { ApiService } from '../../services/api.service';
+import { HasRoleDirective } from '../../_directives/hasRole.directive';
```

Comment:
unused import

Reply:
I think the tests were failing if I didn't add it, but will double-check. I had to jump through a few hoops to get them passing after adding the directive.